app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -44,6 +45,12 @@ def index
 
     private
 
+    def block_biometric_requests_in_production
+      if Rails.env.production? && params['biometric_comparison_required'] == 'true'
```

Comment:
I think this should probably be based on whether `doc_auth_selfie_capture_enabled` is currently true, rather than checking `Rails.env.production?`. I believe `Rails.env.production?` will be true in dev.

Reply:
Yep, it will be true in the sandbox dev environment. We have the ability to check for prod prod though. If needed, I'd probably move the logic to the `feature_management` file though.

https://github.com/18F/identity-idp/blob/8a4b1a44a96535a3c77063baeef4870e36dbc91d/lib/feature_management.rb#L67-L69

Reply:
Changed.